tooling-apis/job-distributor/job/job.proto

```diff
@@ -0,0 +1,206 @@
+syntax = "proto3";
+
+package tooling.distributor.job;
+
+import "google/protobuf/timestamp.proto";
+import "job-distributor/shared/ptypes/label.proto";
+
+option go_package = "job-distributor/job";
+
+// JobService provides RPCs for managing jobs and job proposals within the
+// system.
+service JobService {
```

Comment:
What are the lifecycles of Jobs and Job proposals? Should they have full CRUDL handler implementations?